File: app/src/main/java/com/nataliafavero/santander/data/model/ScreenResponse.java
package com.nataliafavero.santander.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by nataliafavero on 12/09/18.
 */

public class ScreenResponse {

    @SerializedName("screen")
    @Expose
    private Screen screen;

    public Screen getScreen() {
        return screen;
    }
}

File: app/src/main/java/com/vctapps/santanderchallenge/asset/data/entity/ScreenResponse.kt
package com.vctapps.santanderchallenge.asset.data.entity

import com.google.gson.annotations.SerializedName

data class ScreenResponse(@SerializedName("title")      val title: String?,
                          @SerializedName("fundName")   val assetName: String?,
                          @SerializedName("whatIs")     val whatIs: String?,
                          @SerializedName("definition") val definition: String?,
                          @SerializedName("riskTitle")  val riskTitle: String?,
                          @SerializedName("risk")       val risk: Int?,
                          @SerializedName("infoTitle")  val infoTitle: String?,
                          @SerializedName("moreInfo")   val moreInfo: MoreInfoResponse?,
                          @SerializedName("info")       val info: MutableList<InfoDetailResponse>?,
                          @SerializedName("downInfo")   val downInfo: MutableList<InfoDetailResponse>?)